test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -0,0 +1,94 @@
+﻿Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
```

Comment:
@Francisco-Gamino is almost ready on the fix of #951. This test will be updated in the PR for that issue.